### ISSUE TITLE
On Course Un-enrollment, have the option for the User to be unsubscribed

### ISSUE THREAD

**Issue by d4mation** [realbigplugins/learndash-mailchimp#68]
I don't think having this the default would be preferable in the event that the site admin wants them to stay on the mailing list.

https://secure.helpscout.net/conversation/825451983/1818?folderId=955384

**Comment by brashrebel** [comment]
I agree with your assessment. I also think it would be smarter to use something else like a custom field value showing enrollment status or group membership. You could then segment your list based on enrollment. And unsubscribing would never make sense for sites with multiple courses.

**Comment by d4mation** [comment]
> unsubscribing would never make sense for sites with multiple courses.

Ah, yes, I meant remove from the Tag for the Course. That's what I had said in the ticket I believe.
